packages/components/src/font-size-picker/index.tsx

```diff
@@ -298,42 +304,45 @@ const UnforwardedFontSizePicker = (
 								</FlexItem>
 								{ withReset && (
 									<FlexItem isBlock>
-										<Button
+										<ResetButton
 											className="components-color-palette__clear"
```

Comment:
It looks like this classname isn't used anywhere in the repo (plus, it should have a `components-font-size-picker` namespace anyway) — I believe we're safe in deleting it

```suggestion
```